webview/src/setup/components/remotes/CloudVersioning.tsx

```diff
@@ -7,7 +7,7 @@ export const CloudVersioning: React.FC<PropsWithChildren> = ({ children }) => (
   <>
     <InfoText>
       <CloudVersioningLink /> requires {children}
```

Comment:
`children` should probably be in a styled `span` that fixes the formatting problem rather than inserting newlines. Or maybe `To enable run:` should be inside its own `p` element.

Reply:
I missed a few `\n` in the previous review as well. We should not rely on them for styling / formatting, but rather use  tags and css to do so.